### ISSUE TITLE
Crash report

### ISSUE THREAD

**Issue by UncleSpock** [Boomflex/JustNotEnoughMods2#15]
Crash when exploring open ocean
[crash-2020-03-03_12.58.42-server.txt](https://github.com/Boomflex/JustNotEnoughMods2/files/4283335/crash-2020-03-03_12.58.42-server.txt)

**Comment by UncleSpock** [comment]
Chrashed while exploring world with jetpack

[crash-2020-03-03_15.06.54-server.txt](https://github.com/Boomflex/JustNotEnoughMods2/files/4283860/crash-2020-03-03_15.06.54-server.txt)

**Comment by NinjaHJM** [comment]
Hopefully it will help your issue:

I ran into similar problems with a server I'm running. We updated from an existing world to the current version of the pack. The Dungeon Crawl mod was removed from the pack but was still part of the old world file. Downloading the mod into the server and client got it working again.

https://www.curseforge.com/minecraft/mc-mods/dungeon-crawl/files

You'll need to reload the server and client. If you just copy the file over without reloading the client you'll get sync errors.
